### PR TITLE
Fix/sdl 0189 new fixes for review

### PR DESCRIPTION
**Pull request by LiliiaShlikhtLuxoft**
Fixed - minor changes (5 comments from review)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -130,8 +130,9 @@ class HMIApp extends React.Component {
         if (resPeriod > OUT_OF_BOUND_RESET_PERIOD) {
             store.dispatch(resetTimeout(10000));
         } else {
-            store.dispatch(resetTimeout(event.target.value));
+            store.dispatch(resetTimeout(resPeriod));
         }
+        console.log(store.getState())
```
Comment (ValeriiMalkov) [inline src/index.js:135]: Any reason for `console.log` ?

File: src/js/Controllers/TTSController.js [hunk 2]
```diff
@@ -131,9 +131,8 @@ class TTSController {
                 // in case more things will be added to the filePlaylist,
                 // so that TTS Speak will be replied to before speaking next message
                 clearInterval(this.timers[file.id]);
-                this.speakEnded();
                 this.listener.send(RpcFactory.TTSSpeakSuccess(file.id));
-                return;
+                return this.playNext();
```
Comment (ValeriiMalkov) [inline src/js/Controllers/TTSController.js:135]: Did you test these changes?

File: src/index.js (hunk 1 above)
Comment (LiliiaShlikhtLuxoft) [reply src/index.js]: @ValeriiMalkov fixed

File: src/js/Controllers/TTSController.js (hunk 2 above)
Comment (LiliiaShlikhtLuxoft) [reply src/js/Controllers/TTSController.js:135]: @ValeriiMalkov I don't know how to test this changes. Could you help please?

**Pull request merged**